components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/DataFlowProcessingEngine.cs

```diff
@@ -163,124 +140,94 @@ public class DataFlowProcessingEngine : IProcessingEngine, IDisposable {
             diagInfo.AppendLine("  # Ingress ValueChanges (from OPC)    : {valueChangesCount,14:n0} {valueChangesPerSecFormatted}");
 
-            diagInfo.AppendLine("  # Ingress BatchBlock buffer size     : {batchDataSetMessageBlockOutputCount,14:0}");
-            diagInfo.AppendLine("  # Encoding Block input/output size   : {encodingBlockInputCount,14:0} | {encodingBlockOutputCount:0}");
-            diagInfo.AppendLine("  # Encoder Notifications processed    : {notificationsProcessedCount,14:n0}");
-            diagInfo.AppendLine("  # Encoder Notifications dropped      : {notificationsDroppedCount,14:n0}");
-            diagInfo.AppendLine("  # Encoder IoT Messages processed     : {messagesProcessedCount,14:n0}");
-            diagInfo.AppendLine("  # Encoder avg Notifications/Message  : {notificationsPerMessage,14:0}");
+            diagInfo.AppendLine("  # Connection retries (OPC Server)    : {connectionRetries,14:0}");
+            diagInfo.AppendLine("  # Ingress batchsize/trigger interval : {batchSize,14:0} | {batchTriggerInterval}");
+            var encodervaluesPerSecFormatted = _messageEncoder.NotificationsProcessedCount > 0 && totalDuration > 0 ? $"({encoderValuesPerSec:#,0.##}/s)" : "";
+            diagInfo.AppendLine("  # Encoder ValueChanges processed     : {notificationsProcessedCount,14:n0} {encodervaluesPerSecFormatted}");
+            diagInfo.AppendLine("  # Encoder ValueChanges dropped       : {notificationsDroppedCount,14:n0}");
+            diagInfo.AppendLine("  # Encoder IoT Messages produced      : {messagesProcessedCount,14:n0}");
+            diagInfo.AppendLine("  # Encoder avg ValueChanges/Message   : {notificationsPerMessage,14:0}");
             diagInfo.AppendLine("  # Encoder avg IoT Message body size  : {messageSizeAverage,14:n0} {messageSizeAveragePercentFormatted}");
-            diagInfo.AppendLine("  # Encoder avg IoT Chunk (4 KB) usage : {chunkSizeAverage,14:0.#}");
-            diagInfo.AppendLine("  # Estimated IoT Chunks (4 KB) per day: {estimatedMsgChunksPerDay,14:n0}");
-            diagInfo.AppendLine("  # Outgress Batch Block buffer size   : {batchNetworkMessageBlockOutputCount,14:0}");
-            diagInfo.AppendLine("  # Outgress input buffer count        : {sinkBlockInputCount,14:n0}");
-
-            string sentMessagesPerSecFormatted = _messageSink.SentMessagesCount > 0 && totalDuration > 0 ? $"({sentMessagesPerSec:0.##}/s)" : "";
-            diagInfo.AppendLine("  # Outgress IoT message count         : {messageSinkSentMessagesCount,14:n0} {sentMessagesPerSecFormatted}");
-            diagInfo.AppendLine("  # Connection retries                 : {connectionRetries,14:0}");
+            diagInfo.AppendLine("  # Encoder avg IoT Unit (4 KB) usage  : {chunkSizeAverage,14:0.#}");
+            diagInfo.AppendLine("  # Estimated IoT Units (4 KB) per day : {estimatedMsgChunksPerDay,14:n0}");
```

Comment:
The documentation mentions "chunks" and also "meter size" for this, not units. Why introduce a new term? (I think it might be confusing)

Reply:
Ah - got it - will change it back.